src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj

```diff
@@ -3,20 +3,30 @@
   <PropertyGroup>
     <Configurations>Debug;Release;Debug-Intrinsics;Release-Intrinsics</Configurations>
     <UseIntrinsics Condition="'$(UseIntrinsics)' == ''">$(Configuration.EndsWith('-Intrinsics'))</UseIntrinsics>
-    
     <TargetFramework Condition="'$(UseIntrinsics)' != 'true'">netstandard2.0</TargetFramework>
     <TargetFrameworks Condition="'$(UseIntrinsics)' == 'true'">netstandard2.0;netcoreapp3.0</TargetFrameworks>
     <IncludeInPackage>Microsoft.ML.CpuMath</IncludeInPackage>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>$(DefineConstants);CORECLR;PRIVATE_CONTRACTS</DefineConstants>
+    <!-- The SDK we currently restore and use has 2.0 as the maximum supported version -->
+    <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
```

Comment:
You shouldn't need this (either here or in the tests).  Instead, anyone who wants to use the `netcoreapp3.0` version needs to install the 3.0 SDK on their machine.

Reply:
This was added in the late afternoon today in an attempt to pass the `VSTS: public-CI` build test below, but was not very successful.

Reply:
Let's chat this morning about it. There may be some tricks we need to do.

Reply:
Removing the `NETCoreAppMaximumVersion` tag from the current green build results in errors in running `build.cmd -Release` from the root of the ML repo.  How should I proceed?